rundeckapp/src/main/groovy/org/rundeck/app/data/providers/GormUserDataProvider.groovy

```diff
@@ -39,6 +40,7 @@ class GormUserDataProvider implements UserDataProvider {
     public static final int DEFAULT_TIMEOUT = 30
     public static final String SESSION_ABANDONED_MINUTES = 'userService.login.track.sessionAbandoned'
     public static final String SHOW_LOGIN_STATUS = 'gui.userSummaryShowLoginStatus'
+    public static final String NAME_CASE_SENSITIVE_ENABLED = "login.nameCaseSensitiveEnabled"
```

Comment:
if you define a new system configuration key, you should use the `SystemConfigurable` mechanism